src/renderer/pages/vpn.vue

```diff
@@ -158,12 +171,40 @@ export default {
     onCountriesUpdate (countries) {
       this.countriesAreLoading = false
 
-      if (countries.length < 1) this.bugReporter.captureInfoMessage('Renderer received empty countries list')
+      if (countries.length < 1) {
+        this.bugReporter.captureInfoMessage('Renderer received empty countries list')
+      }
 
       this.countryList = countries
+    },
+    onTabClick (page) {
+      if (page !== 'provider') {
+        return
+      }
+
+      if (this.statusCode > -1) {
```

Comment:
Why do we need `statusCode`? It's very hard to understand the intent. We already have `ConnectionStatus` enum somewhere for that.

```suggestion
      if (this.status != ConnectionStatus.NotConnected) {
```